src/w3d/renderer/boxrobj.h

```diff
@@ -20,6 +20,9 @@
 #include "vector3.h"
 #include "w3d_file.h"
 #include "w3dmpo.h"
+#include <cstring>
+
+using std::strcpy;
```

Comment:
never put using statements in headers you are just negating the whole reason a namespace exists.